guide/src/documentation/configuration/filter-incentivized.md

```diff
@@ -0,0 +1,45 @@
+# Filter incentivized packets
+
+Hermes can be configured in order to only relay packets which are incentivized. This is done by using the `[[chain.packet_filter.min_fee]]` configuration.
+
+When this filter is configured, Hermes will not relay ignore `send_packet` events when they do not meet the configured requirements. This configuration can be set to be per channel or for a set of channels using regular expression.
+
+> __WARNING__: This configuration is experimental. Packet clearing will be disabled for the channels which have a fee filter configured, and some `send_packet` events might not be relayed if the incentivized event is not in the same batch of events.
+
+## Examples
+
+___Channel, amount and denom specific___
+
+This example will configure Hermes so it will ignore `send_packet` events from `channel-0` which do not have at least `10 uatoms` as the `recv_fee`.
+
+```
+[chains.packet_filter.min_fees.'channel-0']
+  recv    = [{ amount = 10, denom = 'uatom' }]
+```
+
+___Amount and denom specific___
+
+This example will configure Hermes so it will ignore `send_packet` events from any channel which do not have at least `10 uatoms` as the `recv_fee`.
+
+```
+[chains.packet_filter.min_fees.'*']
+  recv    = [{ amount = 10, denom = 'uatom' }]
+```
+
+___Amount only___
+
+This example will configure Hermes so it will ignore `send_packet` events from any channel which do not have at least 10 of any token as the `recv_fee`.
+
+```
+[chains.packet_filter.min_fees.'*']
+  recv    = [{ amount = 10 }]
```

Comment:
```suggestion
  recv = [{ amount = 10 }]
```

Reply:
I don't know if this is a really useful example, let's use `amount = 0` instead and explain that it will only relay incentivized packets, whatever the fees are.

Reply:
I agree, I will update this to `amout = 0`.